docs/frameworks/laravel.md

```diff
@@ -51,13 +61,13 @@ functions:
 
 Now we still have a few modifications to do on the application to make it compatible with AWS Lambda.
 
-Since [the filesystem is readonly](/docs/environment/storage.md) except for `/tmp` we need to customize where the cache files are stored. Add this line in `bootstrap/app.php` after `$app = new Illuminate\Foundation\Application`:
+Since [the filesystem is readonly](/docs/environment/storage.md) except for `/tmp` we need to customize where the cache files are stored. Change the line in `bootstrap/app.php` at `$app = new Illuminate\Foundation\Application(`:
 
-```php
-/*
- * Allow overriding the storage path in production using an environment variable.
- */
-$app->useStoragePath($_ENV['APP_STORAGE'] ?? $app->storagePath());
+```diff
+$app = new Illuminate\Foundation\Application(
+-    realpath(__DIR__.'/../')
++    $_ENV['APP_BASE_PATH'] ?? dirname(__DIR__)
+);
```

Comment:
It seems it's already the content of the file:

https://github.com/laravel/laravel/blob/c78a1d8184f00f8270d8a135cc21590837b6bfbd/bootstrap/app.php#L14-L16

Are you sure this step is needed now?

Reply:
I’m not sure actually. I was by the understanding that we need the APP_BASE_PATH

Reply:
We need it, but it's already that code in Laravel.

What I am saying is that you shouldn't have to change anything, Laravel is already like this: https://github.com/laravel/laravel/blob/c78a1d8184f00f8270d8a135cc21590837b6bfbd/bootstrap/app.php#L14-L16